components/style/properties/longhand/font.mako.rs

```diff
@@ -344,6 +344,73 @@ ${helpers.single_keyword("font-variant",
     }
 </%helpers:longhand>
 
+<%helpers:longhand products="gecko" name="font-synthesis" animatable="False">
```

Comment:
Hmm... Yeah, I thought you'd also implement the Gecko bits. If you're interested I can guide you in how to do so. Otherwise we need to land it as disabled for now, which isn't great, but is doable.